cmd/kubelet/app/server.go

```diff
@@ -1257,16 +1259,23 @@ func parseResourceList(m map[string]string) (v1.ResourceList, error) {
 	rl := make(v1.ResourceList)
 	for k, v := range m {
 		switch v1.ResourceName(k) {
+		case nodev1.ResourceSwap:
+			if !utilfeature.DefaultFeatureGate.Enabled(features.NodeSwap) {
+				return nil, fmt.Errorf("cannot reserve swap resource as feature gate NodeSwap is disabled")
```

Comment:
```suggestion
				return nil, fmt.Errorf("cannot reserve quantity %q of %q resource as feature gate NodeSwap is disabled", v, k)
```
is there an err we can use here for unsupported/diabled?

Reply:
As we should include that the Node FG is disabled here, a common err cannot do that and I don't find one. 

I updated with your suggested change.